collectors/python.d.plugin/slurm/slurm.chart.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Description: slurm netdata python.d module
+# Author: Max Berrendorf (mberr)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from bases.FrameworkServices.ExecutableService import ExecutableService
+
+SQUEUE_COMMAND = "squeue -o %all"
```

Comment:
comment: this will retrieve all information about running jobs, even though we are currently only using a single column.

On our system, the header looks as follows
```
ACCOUNT|TRES_PER_NODE|MIN_CPUS|MIN_TMP_DISK|END_TIME|FEATURES|GROUP|OVER_SUBSCRIBE|JOBID|NAME|COMMENT|TIME_LIMIT|MIN_MEMORY|REQ_NODES|COMMAND|PRIORITY|QOS|REASON||ST|USER|RESERVATION|WCKEY|EXC_NODES|NICE|S:C:T|JOBID|EXEC_HOST|CPUS|NODES|DEPENDENCY|ARRAY_JOB_ID|GROUP|SOCKETS_PER_NODE|CORES_PER_SOCKET|THREADS_PER_CORE|ARRAY_TASK_ID|TIME_LEFT|TIME|NODELIST|CONTIGUOUS|PARTITION|PRIORITY|NODELIST(REASON)|START_TIME|STATE|UID|SUBMIT_TIME|LICENSES|CORE_SPEC|SCHEDNODES|WORK_DIR
```

We can also restrict the output to the information we are currently collecting by setting it to `squeue -o %T`.